Enterprise-CMCS/macpro-mdct-mfp
# Add create SAR e2e test

### Description

- Add Playwright test for state user creating a SAR
- Refactor `yarn db:seed` script so that seeds can be used with tests
- Add ability to specify year and period in seed script


### Related ticket(s)


CMDCT-3863

---
### How to test

1. cd `tests` and `yarn install` to update packages
2. cd back to root folder,  run `yarn test:e2e tests/e2e/sar.spec.ts` or `yarn test:e2e-ui tests/e2e/sar.spec.ts` for interactive test
3. Verify that e2e test passes
4. Start MFP locally
5. Run `yarn db:seed` and verify script still seeds local database






---
### Pre-review checklist
